cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -141,6 +141,15 @@
   private static final String HIVE_METASTORE_TOKEN_KEY = "hive.metastore.token.signature";
   public static final String SPARK_YARN_DIST_FILES = "spark.yarn.dist.files";
 
+  private static final String PARAMS_EXPLORE_MODIFIES = com.google.common.base.Joiner.on("|").join(
+    ImmutableList.of("spark.*", "explore.*",
+                     "mapreduce.job.queuename",
+                     "mapreduce.job.complete.cancel.delegation.tokens",
+                     "mapreduce.job.credentials.binary",
+                     "hive.exec.submit.local.task.via.child",
+                     "hive.exec.submitviachild",
+                     CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION));
```

Comment:
are these the only ones?

On my cluster I had to add hive.lock.*, and if TEZ is enabled, tex.credentials.path